RetailCoder.VBE/UI/RubberduckUI.fr.resx

```diff
@@ -1305,16 +1069,16 @@ Support localisation WPF par Grant Frisken</value>
     <value>Clé invalide</value>
   </data>
   <data name="AboutWindow_Copyright" xml:space="preserve">
-    <value>© Copyright 2014-2017 Mathieu Guindon &amp; Christopher McClellan</value>
+    <value>© Copyright 2014-2018 Mathieu Guindon &amp; Christopher McClellan</value>
```

Comment:
IIRC there was a PR that made the copyright notice more dynamic. This key might be stale.